README.md

```diff
@@ -400,6 +400,27 @@ Note: To use network policies, you must be using a networking solution that supp
 
 Use NetworkPolicy to control network traffic flow for your applications, it allows you to specify how pods should communicate over the network. By default network policy is disabled in the values.yaml file. Set the networkPolicy.enabled to true to enable the use of network policy resource, default ports are provided in the settings, you can define custom rules for the sources of the traffic to the desired ports.
 
+## Pod Priorty
+
+Pods can be assigned priority that reflects the signifance of a pod compared to other pods. For detailed explanation, please refer [https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/)
```

Comment:
I think we need a bit more detail on why you would set the priority for a MarkLogic pod. Is there a priority that we recommend?

Reply:
Added changes in the fork branch PR: https://github.com/marklogic/marklogic-kubernetes/pull/79